lib/config-validator.ts

```diff
@@ -112,12 +113,19 @@ type PackageJson = {
       ) as PackageJson;
       if (pkgJson.renovate) {
         logger.info(`Validating package.json > renovate`);
-        await validate('package.json > renovate', pkgJson.renovate, strict);
+        await validate(
+          false,
+          'package.json > renovate',
+          pkgJson.renovate,
+          strict,
+          false
+        );
       }
       if (pkgJson['renovate-config']) {
         logger.info(`Validating package.json > renovate-config`);
         for (const presetConfig of Object.values(pkgJson['renovate-config'])) {
           await validate(
+            true,
```

Comment:
Are you sure this is global config? It's a preset isn't it?

Reply:
Sorry, it is a mistake.

Reply:
We need a test file for the config validator file, it will catch these mistakes.